scripts/install-tests.sh

```diff
@@ -13,7 +13,11 @@ fi
 if [ "${TESTING_PROFILE}" = "regression-integration-tests" ]; then
     pip3 install -r dockstore-webservice/src/main/resources/requirements/1.13.0/requirements3.txt
 elif [ "${TESTING_PROFILE}" == "language-parsing-tests" ]; then
-    pip3 install -r dockstore-webservice/src/main/resources/requirements/1.14.0/requirements3.txt
+    #pip3 install -r dockstore-webservice/src/main/resources/requirements/1.14.0/requirements3.txt
```

Comment:
Any reason to keep this line around (commented), rather than delete it completely?

Reply:
Will change to code comment to explain connection